src/app/qgstipfactory.cpp

```diff
@@ -97,19 +97,18 @@ QgsTipFactory::QgsTipFactory() : QObject()
                         " move them around with the arrow keys. For accurate positioning use the"
                         " <strong>Position and Size</strong> section, which can be found in the"
                         " tab <strong>Item Properties -> Position and Size</strong>."
-                        " The other move tool (map canvas icon with right arrow) allows one"
-                        " to move the map content within a map frame."
+                        " The other move tool (a scrolled map icon with four oriented arrows)"
+                        " allows one to move the map content within a map frame."
```

Comment:
Is *allows one* a proper English expression? I have a feeling that this is a literal translation from German. CC @NathanW2 @nyalldawson